packages/flutter/test/material/input_decorator_test.dart

```diff
@@ -1049,6 +1049,49 @@ void main() {
     );
   });
 
+  testWidgets('InputDecorator.debugDescribeChildren', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      buildInputDecorator(
+        decoration: const InputDecoration(
+          icon: const Text('icon'),
+          labelText: 'label',
+          hintText: 'hint',
+          prefixText: 'prefix',
+          suffixText: 'suffix',
+          prefixIcon: const Text('prefixIcon'),
+          suffixIcon: const Text('suffixIcon'),
+          helperText: 'helper',
+          counterText: 'counter',
+        ),
+        child: const Text('text'),
+      ),
+    );
+
+    final RenderObject renderer = tester.renderObject(find.byType(InputDecorator));
+    final List<String> nodeNames = renderer.debugDescribeChildren()
+      .map((DiagnosticsNode node) => node.name)
+      .toList();
+    nodeNames.sort();
+    expect(nodeNames, <String>[
```

Comment:
nit: instead of sorting you could just use one of the `unorderedEquals` or `unorderedMatches` matchers.

Reply:
I didn't know of unorderedEquals, that's pretty slick.